.circleci/config.yml

```diff
@@ -66,6 +66,13 @@ jobs:
       - attach_workspace:
           at: .
       - run: yarn lint
+  verify:
+    docker:
+      - image: circleci/python:3-node
```

Comment:
Any idea how to get a docker image with python3, java, and node? I couldn't find one, at a cursory glance.

Reply:
I couldn't find one either (at least among pre-built ones). According to CircleCI docs you could use [custom docker image](https://circleci.com/docs/2.0/custom-images/), but that seems rather complicated.